profiler/profile.go

```diff
@@ -12,6 +12,9 @@ import (
 	"io"
 	"runtime/pprof"
 	"time"
+
+	pprofile "github.com/google/pprof/profile"
```

Comment:
@pmbauer forgot to ask: are we cool with this? If yes, should we add it to the `go.mod` file? cc @Kyle-Neale

Reply:
good point; yes I think we should add this to the go.mod file

Reply:
2fceb4b22022827fc4c9bd2309d884046cf74514